docs/installguide/install_all.rst

```diff
@@ -28,22 +28,29 @@ For more advanced use of KA Lite, such as changing the default port, see :ref:`r
 or use the command-line ``kalite`` program, which in typical installations can be found at the path
 ``C:\Python27\Scripts\kalite``. Run ``kalite --help`` for usage info.
 
-.. warning::
-    If you need to download and install contentpacks locally for languages other
-    than English, make sure you are doing it
-    **as the same user that installed KA Lite** in the first place. If you
-    perform the contentpack installation as a different user, some content will
-    not load properly. For downloading and installing content packs for offline
-    methods and automatic deployments, see :ref:`content_pack_retrieve_offline`.
+When you start the KA Lite program, you will find a leaf icon in your task tray.
+Right click on this icon to start/stop the server, open the application in a browser, or set other options:
 
+.. image:: windows_task_tray.png
+    :class: screenshot
+
+By default, you can access KA Lite on the installation computer from the address http://127.0.0.1:8008.
+To access KA Lite from other machines, you will need to connect to the same network as the installation computer and
+access port 8008 using the its IP address.
+For example, if the installation computer has the IP address 192.168.0.104 on your network then you can access it from
+other machines on the same network at the address http://192.168.0.104:8008.
+
+For more advanced use of KA Lite, such as changing the default port, see :ref:`running-ka-lite-with-your-own-settings`
+or use the command-line ``kalite`` program, which in typical installations can be found at the path
+``C:\Python27\Scripts\kalite``. Run ``kalite --help`` for usage info.

```

Comment:
Again, hmm, I think this is happening because you've been working on old versions.